coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/transformers/FromUnixtimeOperatorTransformer.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino.transformers;
+
+import java.util.List;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlLiteral;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import com.linkedin.coral.common.functions.FunctionReturnTypes;
+import com.linkedin.coral.common.transformers.SqlCallTransformer;
+
+import static org.apache.calcite.sql.type.ReturnTypes.*;
+import static org.apache.calcite.sql.type.SqlTypeName.*;
+
+
+/**
+ * This transformer operates on SqlCalls with 'FROM_UNIXTIME(x)' Coral IR function
+ * and transforms it to trino engine compatible function - FORMAT_DATETIME(FROM_UNIXTIME(x)).
+ * For Example:
+ * A SqlCall of the form: "FROM_UNIXTIME(10000)" is transformed to
+ * "FORMAT_DATETIME(FROM_UNIXTIME(10000), 'yyyy-MM-dd HH:mm:ss')"
+ */
+public class FromUnixtimeOperatorTransformer extends SqlCallTransformer {
+
+  private static final String FORMAT_DATETIME = "format_datetime";
+  private static final String FROM_UNIXTIME = "from_unixtime";
+
+  @Override
+  protected boolean condition(SqlCall sqlCall) {
+    return sqlCall.getOperator().getName().equalsIgnoreCase(FROM_UNIXTIME);
+  }
+
+  @Override
+  protected SqlCall transform(SqlCall sqlCall) {
+    SqlOperator formatDatetimeOperator = createSqlOperator(FORMAT_DATETIME, FunctionReturnTypes.STRING);
+    SqlOperator fromUnixtimeOperator = createSqlOperator(FROM_UNIXTIME, explicit(TIMESTAMP));
+
+    List<SqlNode> operands = sqlCall.getOperandList();
+    if (operands.size() == 1) {
+      return formatDatetimeOperator.createCall(SqlParserPos.ZERO,
+          fromUnixtimeOperator.createCall(SqlParserPos.ZERO, operands.get(0)),
+          SqlLiteral.createCharString("yyyy-MM-dd HH:mm:ss", SqlParserPos.ZERO));
+    } else if (operands.size() == 2) {
+      return formatDatetimeOperator.createCall(SqlParserPos.ZERO,
+          fromUnixtimeOperator.createCall(SqlParserPos.ZERO, operands.get(0)), operands.get(1));
```

Comment:
operand count is also validated from the function signature defined in the [Registry](https://github.com/linkedin/coral/blob/6a5d8f229d809856efebe5ab93da972147732f7b/coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java#L338). If the operand count is not 2, it should fail prior to this step.

Reply:
From the existing UT in [HiveToTrinoConverterTest](https://github.com/linkedin/coral/blob/6a5d8f229d809856efebe5ab93da972147732f7b/coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/HiveToTrinoConverterTest.java#L374), 
input SQL: `from_unixtime(10000, 'yyyy-MM-dd')` is converter to output SQL: `format_datetime(from_unixtime(10000), 'yyyy-MM-dd')`. So we can keep this transformation since the inner SqlCall is converted from 2 arg sqlCall to 1 arg SqlCall.